source/common/http/filter_manager.cc

```diff
@@ -777,10 +777,15 @@ void FilterManager::decodeTrailers(ActiveStreamDecoderFilter* filter, RequestTra
 }
 
 void FilterManager::decodeMetadata(ActiveStreamDecoderFilter* filter, MetadataMap& metadata_map) {
+  ScopeTrackerScopeState scope(&*this, dispatcher_);
+  filter_manager_callbacks_.resetIdleTimer();
```

Comment:
IMO this is worth calling out in release notes as a functional change / bugfix

Reply:
Done.